be/src/formats/parquet/scalar_column_reader.cpp

```diff
@@ -18,10 +18,32 @@
 #include "gutil/casts.h"
 #include "io/shared_buffered_input_stream.h"
 #include "simd/simd.h"
+#include "statistics_helper.h"
 #include "utils.h"
 
 namespace starrocks::parquet {
 
+Status FixedValueColumnReader::row_group_zone_map_filter(const std::vector<const ColumnPredicate*>& predicates,
```

Comment:
If this is just used for zonemap filter, do you think we can extend the predicate tree to support fixed value expr? just a suggestion.